server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/MapStringTermsAggregator.java

```diff
@@ -148,15 +170,23 @@ LeafBucketCollector getLeafCollector(
      * Fetch values from a {@link ValuesSource}.
      */
     public static class ValuesSourceCollectorSource implements CollectorSource {
-        private final ValuesSource valuesSource;
+        private final ValuesSourceConfig valuesSource;
```

Comment:
Why not rename the variable as well?

Reply:
Because I wasn't being careful. I'll do it.